### DISCUSSION TITLE
Github Education Verification

### DISCUSSION THREAD

**Discussion by zaidamin678** [community/community#58661] [GitHub Education]
### Hi everyone,

Hey,

I am writing this to enquire about facing a verification issue in Github Education Sign-Up.  I received several mails that my account is not verified. I did all the things that are mentioned in Doc for setting hub GitHub Profile.

**Comment by the-officialjosh** [comment]
After submitting your application, please be aware that you will need to wait for a confirmation email. The processing time for receiving this confirmation email typically ranges from 2 days to 7 days. Kindly note that the exact duration may vary depending on various factors. It is recommended to patiently wait for the confirmation email during this period.

**Comment by samus-aran** [comment]
Hi @zaidamin678  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
